server/services/database/applicantProfile.js

```diff
@@ -63,12 +65,13 @@ const createApplicantProfile = ({
   city,
   cvLink,
   rightToWork,
+  profile_picture,
```

Comment:
This is should be: `profilePicLink`

Reply:
in the database part I did it as profile_picture because it may be in the future something different so we don't need to change the names

Reply:
Just because you passed it as `profilePicLink` from you API endpoint. so use one of them, it is not working like this at the moment.

Reply:
### done